aptos-move/aptos-vm/src/automated_transaction_processor.rs

```diff
@@ -33,6 +33,7 @@ use std::ops::Deref;
 
 pub struct AutomatedTransactionProcessor<'m> {
     aptos_vm: &'m AptosVM,
+    for_system_automated_transaction: bool,
```

Comment:
```suggestion
    sys_automated_tx: bool,
```
Just an optional suggestion if we should use shorter names which still conveys the meaning

Reply:
Done